e2e/serial/dappInteractions/3_dappAccountsSwitcher.test.ts

```diff
@@ -42,375 +42,389 @@ let driver: WebDriver;
-    await findElementByTestIdAndClick({ id: 'switch-wallet-item-2', driver });
-    // switch network
-    await findElementByTestIdAndClick({ id: 'switch-network-menu', driver });
-    await findElementByTestIdAndClick({
-      id: `switch-network-item-${ChainId.mainnet}`,
-      driver,
-    });
-
-    await delayTime('medium');
-    await clickAcceptRequestButton(driver);
-
-    await driver.switchTo().window(dappHandler);
-    const topButton = await querySelector(
-      driver,
-      '[data-testid="rk-account-button"]',
-    );
-
-    expect(topButton).toBeTruthy();
-    await waitAndClick(topButton, driver);
-
-    const ensLabel = await querySelector(driver, '[id="rk_profile_title"]');
-    expect(ensLabel).toBeTruthy();
-  });
-
-  it('should be able to go back to extension, switch account and connect from nudge sheet', async () => {
-    await switchWallet(TEST_VARIABLES.SEED_WALLET.ADDRESS, rootURL, driver);
-    await delayTime('long');
-    const appConnectionNudgeSheet = await findElementByTestId({
-      id: 'app-connection-nudge-sheet',
-      driver,
-    });
-    expect(appConnectionNudgeSheet).toBeTruthy();
-    await findElementByTestIdAndClick({ id: 'nudge-sheet-connect', driver });
-    await findElementByTestIdAndClick({ id: 'home-page-header-left', driver });
-    await findElementByTestIdAndClick({
-      id: 'app-connection-menu-connected-apps',
-      driver,
-    });
-    const appConnectionRow = await findElementByTestId({
-      id: `connected-app-bx-test-dapp.vercel.app-${shortenAddress(
-        TEST_VARIABLES.SEED_WALLET.ADDRESS,
-      )}`,
-      driver,
-    });
-    expect(appConnectionRow).toBeTruthy();
-  });
-
-  it('should be able to go back to extension, switch account and connect from nudge banner', async () => {
-    await switchWallet(
-      TEST_VARIABLES.PRIVATE_KEY_WALLET_3.ADDRESS,
-      rootURL,
-      driver,
-    );
-    await delayTime('long');
-    await delayTime('medium');
-    const appConnectionNudgeBaner = await findElementByTestId({
-      id: 'app-connection-nudge-banner',
-      driver,
-    });
-    expect(appConnectionNudgeBaner).toBeTruthy();
-    await findElementByTestIdAndClick({ id: 'nudge-banner-connect', driver });
-
-    await findElementByTestIdAndClick({ id: 'home-page-header-left', driver });
-    await findElementByTestIdAndClick({
-      id: 'app-connection-menu-connected-apps',
-      driver,
-    });
-    const appConnectionRow = await findElementByTestId({
-      id: `connected-app-bx-test-dapp.vercel.app-${shortenAddress(
-        TEST_VARIABLES.PRIVATE_KEY_WALLET_3.ADDRESS,
-      )}`,
-      driver,
-    });
-    expect(appConnectionRow).toBeTruthy();
-  });
-
-  it('should be able to disconnect', async () => {
-    await findElementByTestIdAndClick({
-      id: `connected-app-bx-test-dapp.vercel.app-${shortenAddress(
+describe
+  .runIf(browser !== 'firefox')
+  .skip('Dapp accounts switcher flow', () => {
```

Comment:
Any reason we need to skip this suite? I think this is the only coverage of dapp switching nudges